src/spring-cloud/azext_spring_cloud/_validators.py

```diff
@@ -152,11 +152,30 @@ def validate_jvm_options(namespace):
 
 
 def validate_tracing_parameters(namespace):
+    if namespace.disable_app_insights and namespace.disable_distributed_tracing:
+        raise CLIError("Conflict detected: '--disable-app-insights' can not be set with '--disable-distributed-tracing'.")
+    if (namespace.app_insights or namespace.app_insights_key) and namespace.disable_app_insights:
+        raise CLIError("Conflict detected: '--app-insights' or '--app-insights-key'"
+                       "can not be set with '--disable-app-insights'.")
     if (namespace.app_insights or namespace.app_insights_key) and namespace.disable_distributed_tracing:
         raise CLIError("Conflict detected: '--app-insights' or '--app-insights-key'"
                        "can not be set with '--disable-distributed-tracing'.")
     if namespace.app_insights and namespace.app_insights_key:
         raise CLIError("Conflict detected: '--app-insights' and '--app-insights-key' can not be set at the same time.")
+    if namespace.app_insights == "":

```

Comment:
```suggestion
    if not namespace.app_insights:
```

Reply:
"namespace.app_insights" can be null but it can't be an empty string.